pkg/network/usm/monitor_test.go

```diff
@@ -668,17 +668,38 @@ func cleanProtocolMaps(t *testing.T, protocolName string, manager *manager.Manag
 		_, shouldOnlyZero := mapTypesToZero[mapInstance.Type()]
 
 		key := make([]byte, mapInstance.KeySize())
-		value := make([]byte, mapInstance.ValueSize())
+
+		var value any
+		var emptyValue any
+
+		if mapInstance.Type() == ebpf.PerCPUArray {
+			possibleCPUs, err := ebpf.PossibleCPU()
+			if err != nil {
+				panic("failed to get possible CPUs")
```

Comment:
panics are never a good option, I'd prefer for the test to fail, rather can crashing the entire test binary